internal/repository/port_store_inmem_test.go

```diff
@@ -1,28 +1,27 @@
-package adapters_test
+package repository_test
```

Comment:
Why did you choose to have external test packages to the code package? Is this necessary?

Reply:
I don't want to pollute package namespace with tests + I want to be sure public methods work as expected for package consumers and for that unit tests must be external. Fortunately I don't need any access to internal package resources for that this time :)